src/main/java/org/team4159/lib/CsvWriter.java

```diff
@@ -0,0 +1,51 @@
+package org.team4159.lib;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class CsvWriter {
+  private final SimpleDateFormat date_format = new SimpleDateFormat("'-'yyyy-MM-dd-HH-mm'.csv'");
+
+  private File file;
+  private FileWriter writer;
+
+  public CsvWriter(String filename) {
+    final String suffix = date_format.format(new Date());
```

Comment:
this isn't a suffix, and is there any special reason it's final?

Reply:
it is